### PR TITLE
chore: Clean up unused cmd

### LINKED ISSUE #163: Remove all the cmd that will not be used for Phase-2

### PR DESCRIPTION
**Pull request by gitferry**
Closes #163. In particular:

- removed `fpd tx` command set but moved `NewWithdrawRewardCmd` and `NewSetWithdrawAddressCmd` to the root
- removed default value of `--commission-rate` as it is a required flag
- removed rpc `SignMessageFromChainKey` and relevant functionalities
- removed `fpd export-finality-provider`

### CODE REVIEW

File: finality-provider/cmd/fpd/daemon/export.go
Comment (RafilxTenfen) [inline finality-provider/cmd/fpd/daemon/export.go:1]: I would leave the export command

**Review by RafilxTenfen — APPROVED** [review]
utACK

File: finality-provider/cmd/fpd/daemon/export.go
Comment (gitferry) [reply finality-provider/cmd/fpd/daemon/export.go:1]: what would be the use case?

Comment (RafilxTenfen) [reply finality-provider/cmd/fpd/daemon/export.go:1]: Just to see the information based on the finality provider eots-pk, the signing part could be removed

Comment (gitferry) [reply finality-provider/cmd/fpd/daemon/export.go:1]: we have `fpd finality-provider-info [fp-eots-pk-hex]` for that

Comment (RafilxTenfen) [reply finality-provider/cmd/fpd/daemon/export.go:1]: true

**Pull request merged**